src/main/java/gov/cdc/dataingestion/util/AuthUtil.java

```diff
@@ -19,7 +19,13 @@ public class AuthUtil {
 
     public String getResponseFromDIService(AuthModel authModel, String name) {
         try {
-            UsernamePasswordCredentials credentials = new UsernamePasswordCredentials(authModel.getAdminUser(), new String(authModel.getAdminPassword()));
+            UsernamePasswordCredentials credentials;
+            if(name.equals("register")) {
```

Comment:
Is the register response only applicable to the admin user alone?

Reply:
Yes, registration functionality can only be used by the administrator.